src/org/labkey/test/tests/MessagesLongTest.java

```diff
@@ -424,6 +439,37 @@ public void testSteps()
         assertTextPresent("The following new posts were made yesterday");
     }
 
+    @Test
+    public void testDailyDigestMessage()
+    {
+        goToProjectHome();
+        log("Check email preferences");
+        _portalHelper.clickWebpartMenuItem("Messages", true, "Email Preferences");
+        checkCheckbox(Locator.radioButtonByName("notificationType").index(1));
+        clickButton("Update");
+        clickButton("Done");
+
+        SiteNavBar siteNavBar = new SiteNavBar(getDriver());
+        siteNavBar.enterPageAdminMode();
+        log("Customize message board");
+        _portalHelper.clickWebpartMenuItem("Messages", true, "Admin");
+        new AdminPage(getDriver())
+                .includeGroups(true)
+                .save();
+        siteNavBar.exitPageAdminMode();
+
+       goToProjectHome();
+       log("Check message works in Wiki");
+       _portalHelper.clickWebpartMenuItem("Messages", true, "New");
+       new org.labkey.test.pages.announcements.InsertPage(getDriver())
+                .setRenderAs(WikiHelper.WikiRendererType.RADEOX) // "Wiki Page"
+                .setTitle(MSG1_TITLE_2)
+                .setBody("<b>Daily digest message testing with groups enabled</b>")
+                .submit();
+
+        invokeApiAction("home", "announcements", "sendDailyDigest.api", "Failed to send messages daily digest");
+
+    }
```

Comment:
You should check the notification and make sure it looks correct (e.g. includes group info).
`goToEmailRecord()` will get you to the right page.